### PR TITLE
PropTypes and active links fixes

### PR DESCRIPTION
**Pull request by CPatchane**

### CODE REVIEW

File: stylus/components/nav.styl [hunk 1]
```diff
@@ -68,13 +68,11 @@
...
 
-    .active // deprecated
-    .is-active
     &:hover
         color  charcoal-grey
 
-    .active // deprecated
-    .is-active
+    &.active // deprecated
+    &.is-active
```
Comment (GoOz) [inline stylus/components/nav.styl]: Did you experience any issue to put those `&` back? I removed them on purpose to reduce the weight of selectors.

Comment (ptbrowne) [reply stylus/components/nav.styl]: OK for the fix. For the record, we should use BEM here.

**Comment by GoOz** [comment]
@ptbrowne I asked a question to @CPatchane that wasn't answered… I'd love to have an answer before merging, you know.

**Comment by ptbrowne** [comment]
Sorry @GoOz , I should not have merged, you are right...

File: stylus/components/nav.styl (hunk 1 above)
Comment (CPatchane) [reply stylus/components/nav.styl]: @GoOz Yep, there wasn't any `active` styles anymore on my side

Comment (GoOz) [reply stylus/components/nav.styl]: Oh ok. I didn't get this bug on Drive but i suppose it's done differently on your app, that's why. Ok to re-add it then. For now at least. :)

**Comment by GoOz** [comment]
@ptbrowne no biggie. :)

File: stylus/components/nav.styl (hunk 1 above)
Comment (CPatchane) [reply stylus/components/nav.styl]: Drive seems to use Link component differently indeed, keep me noticed if you have some issues about this point so.

**Pull request merged**